src/components/Audioplayer/ScrollButton/index.js

```diff
@@ -1,9 +1,19 @@
 import React, { PropTypes } from 'react';
+import styled from 'styled-components';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Tooltip from 'react-bootstrap/lib/Tooltip';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-const style = require('../style.scss');
+const ScrollLink = styled.a`
+  width: 100%;
+  display: inline-block;
+  cursor: pointer;
+  padding-right: 1.5%;
+  color: ${props => (props.active ? props.theme.brandPrimary : props.theme.textColor)};
+  outline: none;
+  cursor: pointer;
+  margin-bottom: 0;
+`;
```

Comment:
I don't know, usually in rails we don't like the idea of mixing css/js with html. This approach isn't reusable, can increase response body size and many other small issues. 

Can't we somehow merge all css into one big file and serve it ? this file will be cached so subsequent request would be very fast << Rails way. 

Finally this will inline css for all usage of styles component ? Like [ControlItem](https://github.com/quran/quran.com-frontend/pull/829/files#diff-ac4bb445fc699b6bd56ff5229c476738R39) is used multiple time, each css will be inlined for each `li` ? 
What's impact on rendering performance for this approach ?